app/assets/stylesheets/_buttons.scss

```diff
@@ -26,5 +26,9 @@
   &:disabled {
     cursor: not-allowed;
     opacity: 0.5;
+
+    &:hover {
```

Comment:
This actually _removes_ the hover when the button is `disabled` because the hover on Line 20 still activates.